monai/data/dataset.py

```diff
@@ -1314,22 +1314,23 @@ def __init__(
         col_types: Optional[Dict[str, Optional[Dict[str, Any]]]] = None,
         col_groups: Optional[Dict[str, Sequence[str]]] = None,
         transform: Optional[Callable] = None,
-        **kwargs,
+        **kwargs_read_csv: Optional[Dict] = {},
```

Comment:
thanks, these would be without the `**` and accepting dictionaries, could you please also update the unit tests in `tests/test_csv_dataset.py`? (to run the tests: `python -m tests.test_csv_dataset`)

Reply:
Hi @nvahmadi ,

Could you please help also update CSVIterableDataset?
https://github.com/Project-MONAI/MONAI/blob/dev/monai/data/iterable_dataset.py#L122

Thanks.